src/create-extension-pack.js

```diff
@@ -51,7 +51,7 @@ const repository = 'https://github.com/eclipse-theia/vscode-builtin-extensions';
  * fetched by vscode at build time.
  * https://github.com/microsoft/vscode/blob/1.57.0/product.json#L34-L126
```

Comment:
The reference needs to be updated to 1.64.2

Reply:
Done. I replace the URL with a URI that points to the version of `product.json` that's checked-out locally.